src/components/map/MapService.js

```diff
@@ -392,6 +392,13 @@ goog.require('ga_urlutils_service');
         gaStylesFromLiterals, gaGlobalOptions, gaPermalink,
         gaLang, gaTime) {
 
+      var h2 = function(domainsArray) {
+        if (gaBrowserSniffer.h2) {
+          return domainsArray.slice(1, 2);
+        }
+        return domainsArray;
+      };
```

Comment:
This would add browsersniffer as dependency of urlUtils - not something I'm very fond of.

I propose to keep this little function here, it's really specific to layer/data configuration and is used only in this service. (locality principle)

The tests that I'm going to add have more to do with the h2 detection mechanic in the browsersniffer.

Note: the terrain can not profit from this as the URL's for those are in the layer.json. That's a little sad as both vectortiles and terrain is from cesium - but both resource follow a different paradigm to its configuration.